src/modules/Hosts/Hosts/Strings/en-us/Resources.resw

```diff
@@ -299,6 +299,12 @@
     <value>Entries contain too many hosts</value>
     <comment>"Hosts" refers to the system hosts file, do not loc</comment>
   </data>
+  <data name="EntryAddressIsInvalidWarning.Text" xml:space="preserve">
+    <value>Address not recognized as IPv4 or IPv6</value>
+  </data>
+  <data name="EntryHostsIsInvalidWarning.Text" xml:space="preserve">
+    <value>Invalid host name</value>
```

Comment:
```suggestion
    <value>Must be one or more valid host names separated by spaces</value>
```

Reply:
Grammatical correct I think: `Has to be ...`

Reply:
Actually, on this case it has to be between one and nine, right?